karma.debug.conf.js

```diff
@@ -0,0 +1,16 @@
+// Karma configuration
+
+module.exports = function (config) {
+    'use strict';
+
+    var commonConfig = require('./karma.common.conf');
+
+    commonConfig(config, {
+        singleRun: false,
+        autoWatch: true,
+        browsers: ['Chrome'],
```

Comment:
And indeed, af48d85e74b0a0475e30e11536b3bacfc1dbd238 adds config to run on multiple browsers in Travis.